### ISSUE TITLE
ui5-wizard: javascript error on step quick navigation

### ISSUE THREAD

**Issue by sunnyyadav100** [SAP/ui5-webcomponents#3829]
### **Bug Description**

A user in on Step 2, for quick navigation user click on Step 1. 
After upgrading to RC15, quick navigation is breaking with below javascript error
<img width="1289" alt="image" src="https://user-images.githubusercontent.com/9139037/131975991-58b13cd7-584d-4db9-ae23-68774bbaaca4.png">

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/9139037/131975926-fd08598c-8267-4b0d-8a18-44566384a2e9.png">

### **Expected Behavior**
Quick navigation to step should happen

### **Steps to Reproduce**
2. ...
3. ...

### **Isolated Example**
If possible, please provide an isolated example by forking [CodeSandbox](https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html).

### **Context**
- UI5 Web Components version: {...} RC15
- OS/Platform: {...}
- Browser: {...}
- Affected component: {...}

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...} SAP Ariba
- Bussiness impact: {...}

**Comment by Shtilianova** [comment]
Hello @sunnyyadav100 ,
Please provide an isolated reproducible example via [CodeSandbox](https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html).
Regards, Diana

**Comment by sunnyyadav100** [comment]
The issue is not reproducible with the playground example of Wizard. 
After debugging, the issue is related to `firstFocusableElement` being `null`.  

<img width="1269" alt="image" src="https://user-images.githubusercontent.com/9139037/132033817-47214359-9046-4131-95b4-5afa35f4e74d.png">

Scenario here is If content inside of `ui5-wizard-step` has `0` `offsetWidth` and `offsetHeight` then `firstFocusableElement` is `null`.

<img width="763" alt="image" src="https://user-images.githubusercontent.com/9139037/132034271-da45f695-8a1e-49d1-81bf-e460e83ffe70.png">

We have some component wrappers which has `display: inline; width: auto; height: auto;` (browser computer styles)
If I explicitly set `display: block` on component wrappers then quick navigation is happening without JavaScript error.
But this I need to do for each step, which wasn't the case before upgrading to RC15  (from RC11) - 

```
dms-content-renderer, dms-content-renderer dms-enable-features, dms-content-renderer dms-enable-features dms-field  {
    display: block;
}
```

I think adding a `null` check for `firstFocusableElement` fixes the issue then I don't need to provide custom styles for each step component.

Thanks

**Comment by fifoosid** [comment]
Hi @sunnyyadav100 

Can you please provide the described scenario as an example via [CodeSandbox](https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html).

**Comment by Arora05** [comment]
Hello @fifoosid and @Shtilianova,
I am also trying to implement Wizard but using UI5-webcomponents-react library and I've been able to successfully reproduce this issue in [CodeSandbox](https://codesandbox.io/s/bitter-frog-7xm4w?file=/src/App.js).
The issue is when I try to click on Wizard step "Advanced" in the Sandbox,
The error which comes up is: **TypeError (Cannot read properties of null (reading 'focus'))** 
instead of going in onStepChange function call.
Snippets are:
![Screenshot (228)_LI](https://user-images.githubusercontent.com/89779778/135254131-116aec79-ab0b-4b38-8436-46700c5d5a6c.jpg)
![Screenshot (229)](https://user-images.githubusercontent.com/89779778/135254244-d44e8354-e9f0-42aa-8c78-df77bcb69f15.png)

**Comment by Arora05** [comment]
Hello @fifoosid  and @Shtilianova ,
I am also trying to implement Wizard but using UI5-webcomponents-react library and I've been able to successfully reproduce this issue in [CodeSandbox](https://codesandbox.io/s/bitter-frog-7xm4w?file=/src/App.js)
The issue is when I try to click on Wizard step "Advanced" in the Sandbox,
The error which comes up is: **TypeError (Cannot read properties of null (reading 'focus'))**
instead of going in onStepChange function call.
Snippets are:
![Screenshot (229)_LI](https://user-images.githubusercontent.com/89779778/135254696-0c45f963-9614-4ca8-98d2-9035b16cc6ef.jpg)
![Screenshot (230)](https://user-images.githubusercontent.com/89779778/135254720-b68ba373-a029-4920-9127-ef56f29754ef.png)

**Comment by fifoosid** [comment]
Hi @Arora05 

Thanks for the isolated example!

This issue has been resolved in #3904

**Comment by Arora05** [comment]
Hi @fifoosid,
Can you please tell me the release, in which we expect it to function as expected?

Thanks,
Kshitiz

**Comment by fifoosid** [comment]
Hi @Arora05 

It will be released in 1.0.0-rc.16
